firmware/signing.c

```diff
@@ -658,6 +668,7 @@ static bool signing_sign_hash(TxInputType *txinput, const uint8_t* private_key,
 	}
 	resp.serialized.signature.size = ecdsa_sig_to_der(sig, resp.serialized.signature.bytes);
 
+	uint8_t sighash = signing_hash_type() & 0x1f;
```

Comment:
I think this should be 0xff.  Not that we want to support SIG_ANYONECANPAY soon, but still logically it's the last byte, right?